MeetupPlatformApi/Entities/MeetupEntity.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
 
 namespace MeetupPlatformApi.Entities;
```

Comment:
Please, change position of 'namespace' before 'using'. It's a convention in our project.

Reply:
Fixed.